paper/basic_training.tex

```diff
@@ -154,7 +154,7 @@ \section{Scope of this document}
 
 \section{Science topics}
 \label{sec:science}
-A new practitioner does not have to be an expert in all of the fields that provide the foundation for our simulation methods and analysis of the data produced by these methods. 
+A new simulationist does not have to be an expert in all of the fields that provide the foundation for our simulation methods and analysis of the data produced by these methods. 
```

Comment:
Not sure I like "simulationist" (maybe "simulator", though that also sounds like a device...?). But I tend to prefer "practitioner".

We can leave it for now though.

Reply:
I prefer practitioner too.

Reply:
OK, maybe I'll flip this back before submission. :)

Reply:
I guess the issue I have is that we use that term twice in two sentences back to back. Just read awkwardly to me.

Reply:
Oops, I'd missed that. I'll rephrase.

Reply:
Ah, no, not two sentences back to back. Seems OK.